web/static.go

```diff
@@ -90,6 +90,14 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 		contents = bytes.ReplaceAll(contents, []byte(originalHTML), []byte(modifiedHTML))
 	}
 
+	htmlTemplate := "<html lang=\"%s\">"
+	originalHtmlLocale := fmt.Sprintf(htmlTemplate, html.EscapeString(model.DefaultLocale))
+	modifiedHtmlLocale := getCurrentHtmlTag(c)
+
+	if originalHtmlLocale != modifiedHtmlLocale {
+		contents = bytes.ReplaceAll(contents, []byte(originalHtmlLocale), []byte(modifiedHtmlLocale))
+	}
+
```

Comment:
I wonder if we can do both modifications (this and the `title`) in one passthrough, so we don't end up doing the replace twice